libs/hpc-ui/src/lib/components/form-fields/autocomplete-field.tsx

```diff
@@ -0,0 +1,65 @@
+import { Autocomplete, AutocompleteProps } from '@mui/material';
+import { useField, useFormikContext } from 'formik';
+import tw from 'twin.macro';
+import { StyledTextField } from './text-field';
+import { FormObjectValue } from '@unocha/hpc-data';
+
+const StyledAutocomplete = tw(Autocomplete)`
+      min-w-[10rem]
+      w-full`;
+
+const AutocompleteSelect = ({
+  name,
+  label,
+  options,
+  placeholder,
+  readOnly,
+  isMulti,
+  ...otherProps
+}: {
+  name: string;
+  label: string;
+  options: Array<FormObjectValue>;
+  placeholder?: string;
+  readOnly?: boolean;
+  isMulti?: boolean;
+}) => {
+  const { setFieldValue } = useFormikContext();
+  const [field] = useField<FormObjectValue>(name);
+
+  const configAutocomplete: AutocompleteProps<
+    FormObjectValue,
+    boolean,
+    boolean,
+    boolean
+  > = {
+    ...field,
+    ...otherProps,
+    multiple: isMulti,
+    readOnly,
+    options: options,
```

Comment:
We can use more modern syntax
```suggestion
    options,
```